roles/conda/tasks/main.yml

```
@@ -19,12 +19,38 @@
   loop:
     - conda.sh
     - mamba.sh
-- name: Disable loading base environment
-  command: '{{ conda_root }}/condabin/conda config --set auto_activate_base False --system'
-- name: Get conda environment file
+- name: Install conda merge
+  command: '{{ conda_root }}/condabin/conda install -c conda-forge conda-merge --yes'
```

Mamba should be faster in resolving dependencies, and since we are using mini-forge the conda-forge channel should be the default: 
```suggestion
  command: '{{ conda_root }}/condabin/mamba install conda-merge --yes'
```